### ISSUE TITLE
maildir source: Possible double loading of source message in indexable_chunks

### ISSUE THREAD

**Issue by gauteh** [sup-heliotrope/sup#132]
In message.rb, `indexable_content` does `load_from_source!`. So does the later called `indexable_chunks`/`indexable_body`. `indexable_chunks` may be called independently of `indexable_content` ( #131).

**Comment by danc86** [comment]
I don't think this is an issue anymore. The `indexable_content` method is actually dead code since the Ferret removal in commit 5e1890d982e7cad75beee1c98787ba3fff6376af.
